requirements/edx/base.txt

```diff
@@ -108,12 +108,12 @@ edx-ace==0.1.10
 edx-analytics-data-api-client==0.15.2
 edx-ccx-keys==0.2.1
 edx-celeryutils==0.2.7
-edx-completion==1.0.2
+edx-completion==1.0.3
 edx-django-oauth2-provider==1.3.5
 edx-django-release-util==0.3.1
 edx-django-sites-extensions==2.3.1
 edx-django-utils==1.0.3
-edx-drf-extensions==2.0.1
+edx-drf-extensions==2.1.0
```

Comment:
Hi @jmbowman!  I'd like to bump completion in `ironwood.master` as you did here, to fix the issues with iOS app completion addressed by https://github.com/edx/completion/pull/41.

Is this change suitable for cherry-picking into Ironwood as-is, or do I need to submit a separate PR to just bump `edx-completion` and `edx-drf-extensions==2.1.0`?

Thanks!

Reply:
The latter is probably better; the other libraries jumped enough to probably be an inappropriate patch to a stable release, even if we don't expect it to cause any problems.  The 2 upgrades you want weren't related to the other changes in this PR, they just happened to have been released around that time.